src/android/com/adobe/phonegap/push/PushPlugin.java

```diff
@@ -65,21 +65,7 @@ public void run() {
 
                         Log.v(LOG_TAG, "execute: senderID=" + senderID);
 
-                        String savedSenderID = sharedPref.getString(SENDER_ID, "");
-                        String savedRegID = sharedPref.getString(REGISTRATION_ID, "");
-
-                        // first time run get new token
-                        if ("".equals(savedRegID)) {
-                            token = InstanceID.getInstance(getApplicationContext()).getToken(senderID, GCM);
-                        }
-                        // new sender ID, re-register
-                        else if (!savedSenderID.equals(senderID)) {
-                            token = InstanceID.getInstance(getApplicationContext()).getToken(senderID, GCM);
-                        }
-                        // use the saved one
-                        else {
-                            token = sharedPref.getString(REGISTRATION_ID, "");
-                        }
+                        token = FirebaseInstanceId.getInstance().getToken();
```

Comment:
There are three options : 
 1) Apply the Project Number and Application IDs to firebase manually : https://gist.github.com/secondsun/dfd3a301a5cfbc34d74af18854276f1b
 2) Add to values.xml the following : gcm_defaultSenderId and google_app_id
 3) Include the plugin and the google-services.json file and let them do their magic.